lib/board.rb

```diff
@@ -10,14 +10,24 @@ def combinations
     rows + columns + diagonals
   end
 
+  def full?
+    cells.all?(&:symbol)
+  end
+
+  def available_positions
```

Comment:
What is the difference between a "cell" and a "position"?

Reply:
What I was thinking is that a cell has some sort of state and behavior, whereas a position is just number representing a place that can be moved to on the board.